examples/jsm/loaders/3DMLoader.js

```diff
@@ -616,6 +621,7 @@ class Rhino3dmLoader extends Loader {
 				} else if ( geometry.isLinearLight ) {
 
 					console.warn( 'THREE.3DMLoader:  No conversion exists for linear lights.' );
+					onWarning( 'THREE.3DMLoader:  No conversion exists for linear lights.' );
```

Comment:
What about doing: 
```javascript
self.postMessage( { type: 'warning', 'THREE.3DMLoader:  No conversion exists for linear lights.' } );
```

and adding a warning case (around line 706) where you cal collect warnings?

Reply:
Looks good to me, just need to pass an `id` to be able to call its callback later.